Game/ConsoleApplication2/MovingGameObject.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "GameObject.h"
+#include "Constants.h"
+
+
+class MovingGameObject : public GameObject
+{
+public:
+	// With acceleration
+	MovingGameObject(Rect bounds, Vector2D acceleration, Vector2D maxSpeed, bool isMiniMapObject = false)
+		: GameObject(bounds),
+		m_ACCELERATION(acceleration),
```

Comment:
Hmm, I see your point. Should I change? I guess it's always possible to have global consts (I was after all using some for the Vector directions) and there were quite a few used in SwarmWars